book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] You also need to arrange these arguments to match the Rental `initialize` constructor.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```

Reply:
Thanks for the review @DammyShittu